README.md

```diff
@@ -1 +1,110 @@
-# ansible-role-k8s-web-cluster
+# caktus.k8s-web-cluster
+
+An Ansible role to help set up Kubernetes clusters for web apps.
```

Comment:
"set up" to me sounds as if we might be creating the cluster. Maybe "configure" or some other synonym would be better here?

Reply:
Also maybe we could list what we're doing more specifically, e.g. adding an ingress controller and enabling Let's Encrypt for certificate management.

Reply:
+1 to "configure"